app/models/ranking/report.rb

```diff
@@ -0,0 +1,63 @@
+class Ranking::Report
+  def initialize(site)
+    @site = site
+  end
+
+  def ranked_post_public_ids
+    report_rows.select(&:valid?).sort_by(&:ranking_score).reverse.map(&:public_id)
+  end
+
+  private
+
+  def reporting_api
+    @reporting_api ||= Ranking::ApiClient::ReportingApi
```

Comment:
Don't assign Class to instance var.

https://github.com/bm-sms/daimon-news/pull/694#discussion_r97478124